otter/models/cass.py

```diff
@@ -1789,17 +1794,18 @@ def insert_servers(self, last_update, servers, clear_others):
             queries.append(query.format(cf=self.table, i=i))
         if clear_others:
             return self.delete_servers().on(
-                lambda _: cql_eff(batch(queries), params))
+                lambda _: cql_eff(
+                    batch(queries, self.clock.seconds()), params))
         else:
-            return cql_eff(batch(queries), params)
+            return cql_eff(batch(queries, self.clock.seconds()), params)
```

Comment:
Was there some weirdness with Cassandra and the format of the timestamps needing to be milliseconds?  I think elsewhere we use https://github.com/rackerlabs/otter/blob/master/otter/models/cass.py#L601, but I don't remember the exact details.